src/game_updater/LevelUpdater.cpp

```diff
@@ -1,6 +1,19 @@
 #include "LevelUpdater.h"
 
-LevelUpdater::LevelUpdater() : GameUpdater() {}
+LevelUpdater::LevelUpdater() {
+  this->position = 0;
+}
 LevelUpdater::~LevelUpdater() {}
 
-void LevelUpdater::Update(std::shared_ptr<Level> game_state) {}
+void LevelUpdater::Update(std::shared_ptr<Level> level) {
+  if (position == 0) {
+    position = 1;
+    level->getMusic()->play();
+    level->getMusic()->setLoop(false);
```

Comment:
thats just on the first call to update I added
`#define START 0
#define PLAYING 1
#define END 2
`
latter I'll push that diff